Tests/AK/TestSyncGenerator.cpp

```diff
@@ -106,8 +106,9 @@ TEST_CASE(move_count)
 {
     auto gen = generate2();
     auto result = gen.next();
+    // FIXME: Get this back to down to 3 after the constexpr Optional change, then:
```

Comment:
Maybe also cross-reference this in Optional.h itself

Reply:
Just trying to get CI back to green – feel free to tweak :)